packages/react/src/components/VideoPlayer/VideoPlayer.js

```diff
@@ -55,7 +61,7 @@ const VideoPlayer = ({ inverse, showCaption, videoId, customClassName }) => {
       </div>
       {showCaption && (
         <div className={`${prefix}--video-player__video-caption`}>
-          {videoData.name} {videoDuration}
+          {videoDesc} {videoDuration}
```

Comment:
@annawen1 The caption should actually use `videoData.name`. This was fixed in #2363

Reply:
oh whoops - didn't notice that when i was dealing with the merge conflicts, my bad!